db/schema.rb

```diff
@@ -200,22 +201,6 @@
   add_foreign_key "facts_metrics", "dimensions_dates", primary_key: "date"
   add_foreign_key "facts_metrics", "dimensions_editions"
 
-  create_view "aggregations_search_last_thirty_days", materialized: true,  sql_definition: <<-SQL
```

Comment:
Why do we have changes on the `schema.rb` for the materialized views?

Reply:
I've no idea. Everything is still there it's generated things in a different order.